RecoVertex/Configuration/python/RecoVertex_cff.py

```diff
@@ -36,33 +36,78 @@
 DA2D_vectParameters.TkDAClusParameters.verbose = cms.untracked.bool(False)
-unsortedOfflinePrimaryVertices4D = unsortedOfflinePrimaryVertices.clone( verbose = cms.untracked.bool(False),
+unsortedOfflinePrimaryVertices4DnoPID = unsortedOfflinePrimaryVertices.clone( verbose = cms.untracked.bool(False),
                                                                          TkClusParameters = DA2D_vectParameters )
-unsortedOfflinePrimaryVertices4D.TkFilterParameters.minPt = cms.double(0.0)
-unsortedOfflinePrimaryVertices4D.TrackTimesLabel = cms.InputTag("trackTimeValueMapProducer:generalTracksConfigurableFlatResolutionModel")
-unsortedOfflinePrimaryVertices4D.TrackTimeResosLabel = cms.InputTag("trackTimeValueMapProducer:generalTracksConfigurableFlatResolutionModelResolution")
-offlinePrimaryVertices4D=sortedPrimaryVertices.clone(vertices="unsortedOfflinePrimaryVertices4D", particles="trackRefsForJetsBeforeSorting4D", trackTimeTag=cms.InputTag("trackTimeValueMapProducer","generalTracksConfigurableFlatResolutionModel"),trackTimeResoTag=cms.InputTag("trackTimeValueMapProducer","generalTracksConfigurableFlatResolutionModelResolution"),assignment=dict(useTiming=True))
-offlinePrimaryVertices4DWithBS=sortedPrimaryVertices.clone(vertices="unsortedOfflinePrimaryVertices4D:WithBS", particles="trackRefsForJetsBeforeSorting4D", trackTimeTag=cms.InputTag("trackTimeValueMapProducer","generalTracksConfigurableFlatResolutionModel"),trackTimeResoTag=cms.InputTag("trackTimeValueMapProducer","generalTracksConfigurableFlatResolutionModelResolution"),assignment=dict(useTiming=True))
+unsortedOfflinePrimaryVertices4DnoPID.TkFilterParameters.minPt = cms.double(0.0)
+unsortedOfflinePrimaryVertices4DnoPID.TrackTimesLabel = cms.InputTag("trackExtenderWithMTD:generalTrackt0")
+unsortedOfflinePrimaryVertices4DnoPID.TrackTimeResosLabel = cms.InputTag("trackExtenderWithMTD:generalTracksigmat0")
+trackWithVertexRefSelectorBeforeSorting4DnoPID = trackWithVertexRefSelector.clone(vertexTag="unsortedOfflinePrimaryVertices4DnoPID")
+trackWithVertexRefSelectorBeforeSorting4DnoPID.ptMax=9e99
+trackWithVertexRefSelectorBeforeSorting4DnoPID.ptErrorCut=9e99
+trackRefsForJetsBeforeSorting4DnoPID = trackRefsForJets.clone(src="trackWithVertexRefSelectorBeforeSorting4DnoPID")
+offlinePrimaryVertices4DnoPID=sortedPrimaryVertices.clone(vertices="unsortedOfflinePrimaryVertices4DnoPID", particles="trackRefsForJetsBeforeSorting4DnoPID", trackTimeTag=cms.InputTag("trackExtenderWithMTD","generalTrackt0"),trackTimeResoTag=cms.InputTag("trackExtenderWithMTD","generalTracksigmat0"),assignment=dict(useTiming=True))
+offlinePrimaryVertices4DnoPIDWithBS=sortedPrimaryVertices.clone(vertices="unsortedOfflinePrimaryVertices4DnoPID:WithBS", particles="trackRefsForJetsBeforeSorting4DnoPID", trackTimeTag=cms.InputTag("trackExtenderWithMTD","generalTrackt0"),trackTimeResoTag=cms.InputTag("trackExtenderWithMTD","generalTracksigmat0"),assignment=dict(useTiming=True))
```

Comment:
newlines

It seems also practical to move all the phase-2 timing vertices to their own file.